src/components/structures/auth/ForgotPassword.js

```diff
@@ -117,10 +119,11 @@ module.exports = React.createClass({
         });
     },
 
-    onSubmitForm: function(ev) {
+    onSubmitForm: async function(ev) {
         ev.preventDefault();
 
-        if (!this.state.serverIsAlive) return;
+        const shouldBlockSubmit = await this._checkServerLiveliness(this.props.serverConfig);
+        if (shouldBlockSubmit) return;
```

Comment:
Not sure really what's gained by doing this - shouldn't we just send the request and see if it works?

Reply:
it's the same error with and without the flag, so I'll take it out.